vital/types/color.h

```diff
@@ -13,76 +13,80 @@
 #include <iostream>
 
 namespace kwiver {
+
 namespace vital {
 
-/// Struct to represent an RGB tuple
+// ----------------------------------------------------------------------------
+/// Struct to represent an RGB tuple.
 struct rgb_color
 {
-  /// Default constructor - set the color to white
```

Comment:
I think it's worth keeping a doxygen comment that indicates that the default color is white for the default constructor.

Reply:
Yeah, I was on the fence there anyway, but didn't like the wording... I made another attempt; better?